### ISSUE TITLE
Importing Error - Creating Expansion placeholder

### ISSUE THREAD

**Issue by Joshuak52** [PlaceholderAPI/PlaceholderAPI#871]
### Confirmation

- [X] My issue isn't already found on the Issue tracker.
- [X] My issue is about **PlaceholderAPI** and not any expansion or external plugin
- [X] The issue isn't already fixed in a Spigot Release or Development Build.
- [X] The [Common Issues](https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Common-Issues) page doesn't mention this issue.

### Type

API Bug

### What happens?

When importing PlaceholderAPI  into code I'm getting errors

It's allowing me to import PlaceholderExpansion but the import
import me.clip.placeholderapi.expansion.PlaceholderExpansion;
is throwing error
"The import me cannot be resolved"

https://user-images.githubusercontent.com/6070403/184503561-2f1856d3-aeba-402b-b8d9-00f732866a03.mp4

### Expected Behaviour

I'd hope it just throws no import errors so I could continue to finish the expansion place holder

### How to Reproduce

main.java
```java
package com.heartstealsmp;

import org.bukkit.Bukkit;
import org.bukkit.event.Listener;
import org.bukkit.plugin.java.JavaPlugin;

public class main extends JavaPlugin implements Listener{
	
	@Override
	public void onEnable() {
		Bukkit.getServer().getPluginManager().registerEvents(this, this);
	}
}
```

ExampleExpansion.java
```java
package com.heartstealsmp;


import org.bukkit.entity.Player;

import me.clip.placeholderapi.expansion.PlaceholderExpansion;

public class ExampleExpansion extends PlaceholderExpansion {

    public boolean canRegister() {
        return true;
    }

    public String getAuthor() {
        return "clip";
    }

    public String getIdentifier() {
        return "example";
    }

    public String getPlugin() {
        return null;
    }

    public String getVersion() {
        return "1.0.0";
    }

    public String onPlaceholderRequest(Player p, String identifier) {

        // %example_placeholder1%
        if (identifier.equals("placeholder1")) {
            return "placeholder1 works";
        }
        // %example_placeholder2%
        if (identifier.equals("placeholder2")) {
            return "placeholder2 works";
        }

        return null;
    }
}
```

### `/papi dump` Output

No dump file for this

### Console Log

none yet

### Errors

none yet

**Comment by iGabyTM** [comment]
Do you have the artifact and repo on your pom/build file?

**Comment by Andre601** [comment]
As Gaby pointed out, make sure to properly import the right repository and API version, as explained on the wiki: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Hook-into-PlaceholderAPI#first-steps

**Comment by Joshuak52** [comment]
> As Gaby pointed out, make sure to properly import the right repository and API version, as explained on the wiki: https://github.com/PlaceholderAPI/PlaceholderAPI/wiki/Hook-into-PlaceholderAPI#first-steps

Well how would I go about this though since I'm using eclipse? I was to believe I was able to just add the jar file to the libraries directory and it would work that way?

**Comment by Andre601** [comment]
> Well how would I go about this though since I'm using eclipse? I was to believe I was able to just add the jar file to the libraries/java build path directory and it would work that way?

I'm sure you can google the answer. It's usually better to use maven or gradle to manage dependencies and Eclipse should support both (Maybe through plugins/add-ons)

**Comment by Joshuak52** [comment]
> > Well how would I go about this though since I'm using eclipse? I was to believe I was able to just add the jar file to the libraries/java build path directory and it would work that way?
> 
> I'm sure you can google the answer. It's usually better to use maven or gradle to manage dependencies and Eclipse should support both (Maybe through plugins/add-ons)

No idea why but I fixed this issue by downgrading to version 2.10 of PlaceholderAPI
I guess in the newer versions it only allows importing with maven or gradle
